DFC.ServiceTaxonomy.ApiFunction/CypherQueries/GetJobProfilesBySearchTerm.json

```diff
@@ -1,5 +1,5 @@
 {
-  "query": "MATCH (jp:ncs__JobProfile)-[:ncs__relatedOccupation]-(oc:esco__Occupation) WHERE jp.skos__prefLabel CONTAINS $searchTerm OR oc.ncs__description CONTAINS $searchTerm WITH jp,oc, { ResultItemTitle:jp.skos__prefLabel, ResultItemAlternativeTitle:REDUCE(s = HEAD(oc.skos__altLabel), n IN TAIL( oc.skos__altLabel) | s + ', ' + n), ResultItemOverview:jp.ncs__Description, ResultItemSalaryRange:'�' + jp.ncs__SalaryStarter + ' to ' + '�' + jp.ncs__SalaryExperienced, ResultItemUrlName:$apiHost + '/getjobprofilesbysearchterm/execute/' + jp.ncs__JobProfileWebsiteUrl, JobProfileCategories:['ToDo'] } as jobProfiles WITH { TotalResults:size(collect(jobProfiles)), Profiles:collect(jobProfiles) } AS aggregatedResults UNWIND aggregatedResults.Profiles AS unwoundProfiles WITH unwoundProfiles, aggregatedResults SKIP toInteger(($page-1)*$pageSize) LIMIT toInteger($pageSize) WITH aggregatedResults { PageSize:$pageSize, CurrentPage:$page, Count:aggregatedResults.TotalResults, Results:collect(unwoundProfiles) } AS finalResult RETURN finalResult",
+  "query": "with $searchTerm as lowerlabel call db.index.fulltext.queryNodes(\"JobProfiles\", \"skos__prefLabel:*\"+lowerlabel + \"*\") yield node, score MATCH (node)-[:ncs__relatedOccupation]-(oc:esco__Occupation) WITH node,oc { ResultItemTitle:node.skos__prefLabel, ResultItemAlternativeTitle:REDUCE(s = HEAD(oc.skos__altLabel), n IN TAIL( oc.skos__altLabel) | s + ', ' + n), ResultItemOverview:node.ncs__Description, ResultItemSalaryRange:'�' + node.ncs__SalaryStarter + ' to ' + '�' + node.ncs__SalaryExperienced, ResultItemUrlName:$host + 'getjobprofilesbysearchterm/execute/' + node.ncs__JobProfileWebsiteUrl, JobProfileCategories:['ToDo'] } as jobProfiles WITH { TotalResults:size(collect(jobProfiles)), Profiles:collect(jobProfiles) } AS aggregatedResults UNWIND aggregatedResults.Profiles AS unwoundProfiles WITH unwoundProfiles, aggregatedResults SKIP toInteger(($page-1)*$pageSize) LIMIT toInteger($pageSize) WITH aggregatedResults { PageSize:$pageSize, CurrentPage:$page, Count:aggregatedResults.TotalResults, Results:collect(unwoundProfiles) } AS finalResult RETURN finalResult\"",
```

Comment:
Think you should be able to set the `pageSize` and `page` params to `"type": "System.Int32"` and remove the `toInteger` calls.